index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
 
 This technote describes the details of the Observing Environment and how it is managed by our team.
 
-Add content here
-================
+Introduction
+============
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+The distributed nature of the Vera Rubin Observatory Control System makes it challenging to manage shared application/library software packages between users and the control system.
+In order to mitigate this issue we have created to so called "observing environment".
```

Comment:
I've always referred to it on slack and in conversation as the "Common Observing Environment". I don't think that's any more specific than "observing environment", but worth thinking if we need to have a more descriptive name here as we put this into words....